src/NHibernate/Collection/Generic/PersistentGenericMap.cs

```diff
@@ -22,15 +25,22 @@ namespace NHibernate.Collection.Generic
 	[DebuggerTypeProxy(typeof(DictionaryProxy<,>))]
 	public partial class PersistentGenericMap<TKey, TValue> : AbstractPersistentCollection, IDictionary<TKey, TValue>, ICollection
 	{
-		protected IDictionary<TKey, TValue> WrappedMap;
+		protected IDictionary<TKey, TValue> WrappedMap { get; private set; }
```

Comment:
Protected only for one test file, looks like it should have been private but experienced a probable "rogue" change. At least switched to property with private setter.

Reply:
Can this please be reverted? And the test needs to change to use `.Entries(...)` method instead.